consensus/types/src/beacon_state.rs

```diff
@@ -1321,7 +1428,7 @@ impl<T: EthSpec> BeaconState<T> {
         epoch: Epoch,
         spec: &ChainSpec,
     ) -> Result<Epoch, Error> {
-        Ok(epoch.safe_add(1)?.safe_add(spec.max_seed_lookahead)?)
+        Ok(spec.compute_activation_exit_epoch(epoch)?)
     }
 
     /// Return the churn limit for the current epoch (number of validators who can leave per epoch).
```

Comment:
This function requires the committee caches are built, not the `epoch_cache` correct?